### DISCUSSION TITLE
Why inner child comment is at same level?

### DISCUSSION THREAD

**Discussion by Shubhamsharma585** [posativ/isso#961] [Q&A]
How I can move them to the inner depth? child of child comment should be go more inside

**Comment by BBaoVanC** [comment]
The max nesting level was set to 1 in #79. The reasoning was because of pagination performance, and this article: https://blog.codinghorror.com/web-discussions-flat-by-design/

There was an issue to add configurable nesting level ([#210](https://github.com/posativ/isso/issues/210#issuecomment-1001245602)) but @ix5 closed it, because the nesting is limited by design.
